client/tests/global.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vitest" />
```

Comment:
Maybe we can move this file to the`/types` folder?

Reply:
good idea!